src/lib/Metadata/DomMetadata.cpp

```diff
@@ -871,6 +871,10 @@ DomInfo<T>::construct() const
         auto aliased = domCorpus_.get(I_.AliasedSymbol);
         entries.emplace_back("name", aliased.get("name"));
         entries.emplace_back("symbol", aliased);
+        if (I_.Qualifier)
```

Comment:
This needs to `entries.emplace_back("qualifier", domCreate(I_.Qualifier, domCorpus_));`